Frameworks/CoreText/CTTypesetter.mm

```diff
@@ -116,24 +126,39 @@ static CFIndex _DoWrap(CTTypesetterRef ts, CFRange range, WidthFinderFunc widthF
 
         int curChar = chars[curIndex];
 
-        if (curChar != 10 && curChar != 13 && curChar != 9) {
-            characters.push_back((WORD)curChar);
-        } else {
+        // FT does not seem to be able to print tabs, so we have to replace it with a space for now
+        if (curChar == '\t') {
+            curChar = ' ';
+        }
+
+        if (curChar == 10 || curChar == 13) {
```

Comment:
> 10 || curChar == 13 [](start = 22, length = 20)

magic numbers are no fun. consider using a static const variable at the top with a comment on the value meaning.